fleeto_backend/app/controllers/customer/v1/trips_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Customer::V1::TripsController < Customer::V1::BaseController
+  before_action :set_trip, only: [:show]
+
+  def show
+    render json: @trip, serializer: CustomerTripSerializer
```

Comment:
isnt' it better to have modules customer and driver in serializes folder, and have these serializes nested under it ? so that we stick to our convention,